zproject/prod_settings_template.py

```diff
@@ -209,6 +210,27 @@
 #
 #SOCIAL_AUTH_SUBDOMAIN = 'auth'
 
+########
+# Cognito OAuth.
+#
+# To set up Cognito authentication, you'll need to do the following:
+#
+# (1) Visit https://console.aws.amazon.com/cognito/home , and select
+# "Manager User Pools".
+#
+# (2) Choose an existing pool or create a now one following the Cognito Pool Tutorial
+# https://docs.aws.amazon.com/cognito/latest/developerguide/tutorial-create-user-pool.html
+#
+# (3) Create an app and configure a pool domain
+#
+# (4) You should get a client ID and a client secret. Copy them.
+# Use the client ID as `SOCIAL_AUTH_COGNITO_KEY` here, and put the
+# client secret in zulip-secrets.conf as `social_auth_cognito_secret`.
+#SOCIAL_AUTH_COGNITO_KEY = <your client ID from Cognito>
+#
+# (5) Provide the Cognito domain as SOCIAL_AUTH_COGNITO_POOL_DOMAIN here.
+#SOCIAL_AUTH_COGNITO_POOL_DOMAIN = example.com
```

Comment:
Should we say `https://example.com`? I see that it's configured as `https://example.com` in `test_extra_settings.py` and also do not see python-social-auth prepending `https://`. It would probably be confusing to the admin configuring this if we just say `example.com` and they might just set it like that missing the `https` part.